Tests/UnitTests/Core/Surfaces/SurfaceTests.cpp

```diff
@@ -149,27 +157,27 @@ BOOST_AUTO_TEST_CASE(EqualityOperators) {
   DetectorElementStub detElement1{pTransform1, pPlanarBound, 0.2, pMaterial};
   DetectorElementStub detElement2{pTransform1, pPlanarBound, 0.3, pMaterial};
   DetectorElementStub detElement3{pTransform2, pPlanarBound, 0.3, pMaterial};
-  //
+
   SurfaceStub surface1(detElement1);
   SurfaceStub surface2(detElement1);  // 1 and 2 are the same
   SurfaceStub surface3(detElement2);  // 3 differs in thickness
   SurfaceStub surface4(detElement3);  // 4 has a different transform and id
   SurfaceStub surface5(detElement1);
   surface5.assignSurfaceMaterial(pMaterial);  // 5 has non-null surface material
-  //
+
   BOOST_CHECK(surface1 == surface2);
-  //
-  // remove test for the moment,
-  // surfaces do not have a concept of thickness (only detector elements have)
+
+  // remove test for the moment, surfaces do not have a concept of thickness
+  // (only detector elements have)
   // only thickness is different here
-  //
-  // BOOST_CHECK_NE(surface1, surface3);  // will fail
-  //
+  // BOOST_CHECK_NE(surface1, surface3);
```

Comment:
`// will fail` seems like useful information

Reply:
I enabled the test after all. We can do it now :)